test/helpers.ts

```diff
@@ -43,7 +43,7 @@ export async function waitForUserContact(
 ): Promise<PublicKeyBundle | undefined> {
   return pollFor(
     async () => {
-      const contact = await c1.getUserContactFromNetwork(c2.address)
+      const contact = await c1.getUserContact(c2.address)
```

Comment:
[pebble] This is a substantial change to the ContactBundle tests as we are potentially using the cached value, rather than the freshest bundle on the network. 

While the this will not cause a change in behavior today, when Key Rotation is added there will be a meaningful difference between `GetUserContact` and `GetUserContactFromNetwork`

Will need future consideration

Reply:
I hadn't considered the implications around rotation, but we can always refactor to export the `getUserContactFromNetwork` function on the client and use that in tests if needed. I left it as-is, as it didn't seem like switching to the cached provider had an impact on the current test suite.